webformsasinput.md

```diff
@@ -0,0 +1,133 @@
+
+But we know you did not do that (where is the fun there right?). So lets setup one from scratch.
+
+### General Settings
+
+![Webform General Settings](../imgs/webform-general-settings.jpg)
+Gist here is (look at the screenshot and copy the settings):
+
+- Disable Saving of Form Submissions. You won't need this form to generate a Native Webform Submission entry. 
+- AJAX Settings: check "use ajax". We want people to have the experience of staying in a single page while the create a new ADO and go to, e.g, a Multi Step Webform Workflow.
+
+### Confirmation Settings
+
+![Webform Confirmation Settings](Development/esmero/esmero-docker/release-8.x-1.0-beta1/archipelago-documentation/imgs/webform-confirmation-settings.jpg)
+Gitst here is (again, look at the screenshot and copy the settings):
+- Select "Inline Confirmation". You don't want Webform to send your user to another page while they are still ingesting their ADOs.
+
+### Handler
+![Webform Handler Settings](Development/esmero/esmero-docker/release-8.x-1.0-beta1/archipelago-documentation/imgs/webform-handler-settings.jpg)
+The glue, the piece of resistance. The handler is the one that knows how to talk to a SBF. In simple words, the handler (any handler) provides functionality that does something with a Webform Submission. The one that you want to select here, is the "Strawberryfield harvester" handler. Add it, name it whatever you like (or copy what you see in the screenshot) and make sure you select, if running using our deployment strategy, "S3 File System" as the option for "Permanent Destination for uploaded files". The wording is tricky there, its not really Permantent, since that is handled by Archipelago, but more to Temporary, while working in ingesting an Object, destination for the Webform. Its not really wrong neither. Its permanent for the Webform, but we have better plans for the files and metadata!
+
+Save your settings. And you are ready to roll. That webform can now be used as a Setting for any of the StrawberryField Widgets that use Webforms.
+
+Finally (the real finally). Archipelago encourages at least one Field/JSON key to be present always. The "type" JSON key. So make sure that your Custom Webform has that one, you can copy its config from the provided Webform (in the future we will generate it automatically for you, no worries, we are working to make this easier). having a "type" value will make your life easier. You don't need it, but everything works smoother that way. Since you have a single Content Type named Digital Object, having a "type" Webform field, which leads to a "type" JSON key allows you to discern the Nature of your Digital Object, book or Podcast, Image or 3D and do smart, nice things with them.
```

Comment:
@DiegoPino Do we need here to specify where check if type JSON key is present? Not so clear to me.

Reply:
Also true. I was almost giving up on those last sentences. I feel setting up/creating a webform should almost its own documentation. I will add more details during the morning. Now its a good time to retake your idea of a repository of example webforms. Feels setting some of these things directly on the yaml is faster. Also. I want to code some of these into the handler. Basically settings we need will be forced by the handler. Anyway, you are right !